src/test/java/net/sf/jabref/importer/fetcher/CrossRefTest.java

```diff
@@ -34,14 +35,30 @@ public void findTitleOnly() {
     }
 
     @Test
-    public void findIncompleteTitle() {
+    public void notFindIncompleteTitle() {
```

Comment:
I find the new validity check rather strict. Is CrossRef returning that many false positives?